tools/build/dart.ts

```diff
@@ -0,0 +1,48 @@
+/// <reference path="../typings/node/node.d.ts" />
+
+const which = require('which');
+const child_process = require('child_process');
+
+
+export type DartSdkExecutableMap = {
+  ANALYZER: string,
+  DARTDOCGEN: string,
+  DARTFMT: string,
+  PUB: string,
+  VM: string,
+};
+
+/**
+ * Return a map of TYPE
```

Comment:
Treid to be better about it. In the long term we'd get rid of this file altogether.